rust_snuba/src/processors/spans.rs

```diff
@@ -32,33 +32,33 @@ pub fn process_message(
 }
 
 #[derive(Debug, Default, Deserialize, JsonSchema)]
-struct FromSpanMessage {
-    description: Option<String>,
-    duration_ms: u32,
-    #[serde(alias = "end_timestamp_micro")]
```

Comment:
This alias looks like it disappeared

Reply:
that and the optional on timestamp_precise are both unused I think